datahub-web-react/src/app/ProtectedRoutes.tsx

```diff
@@ -10,8 +10,10 @@ import { HomePage } from '@app/home/HomePage';
 import { HomePage as HomePageV2 } from '@app/homeV2/HomePage';
 import { IntroduceYourself } from '@app/homeV2/introduce/IntroduceYourself';
 import { useSetUserPersona } from '@app/homeV2/persona/useUserPersona';
+import { HomePage as HomePageNew } from '@app/homepageV2/HomePage';
```

Comment:
at least this file path should be `homePageV2` but there might also be a better name for it. i see we have `home` and `homeV2` for alchemy, so we could always just call it `homeV3` to keep consistent with that pattern. In fact yeah i think i'm a fan of that. in a followup we can change that